prestashopv1/plugin/modules/indodana/library/Indodana/Payment/autoload.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+require_once(__DIR__ . DIRECTORY_SEPARATOR . 'vendor' . DIRECTORY_SEPARATOR . 'autoload.php');
+
+define('INDODANA_PLUGIN_ROOT_DIR', __DIR__ . DIRECTORY_SEPARATOR);
+
+// Required for IndodanaLogger
```

Comment:
Can you incorporate `IndodanaLogger` on each important process? We're heavily using this as our log source when the merchant had some problem integrating to our plugin (so that we can see the log there and know what's the problem)

You can refer to any plugin on our repositories to find example usages on critical operation

Reply:
Okay